src/lib.rs

```diff
@@ -347,12 +347,12 @@ impl_arbitrary_for_floats! {
 impl<'a> Arbitrary<'a> for char {
     fn arbitrary(u: &mut Unstructured<'a>) -> Result<Self> {
         use std::char;
-        const CHAR_END: u32 = 0x0011_000;
+        const CHAR_END: u32 = 0x0001_1000;
```

Comment:
Isn't the correct number here `0x11_0000`? cc @fitzgen

The highest Unicode code point is `0x10_FFFF`

Looks like clippy found a bug?

Reply:
I was definitely cribbing from some other code and I guess I mistranslated it, whoops! :grimacing: